packages/tests/README.md

```diff
@@ -0,0 +1,27 @@
+# Tests
+
+The `marble-front/tests` package is designed to provide a comprehensive set of end-to-end tests for the Marble front-end application. By running these tests, developers can ensure that the front-end application is functioning correctly and that any changes or updates to the codebase do not introduce regressions or bugs.
+
+> **Note:** The tests in this package are written using the [Playwright](https://playwright.dev/) testing framework.
+
+## Getting Started
+
+To get started with running the tests in this package, follow these steps:
+
+> **Note:** The tests in this package are designed to be run against a local test environment. Steps 1 and 2 are manual steps that will be automated in the future.
+
+1. Create the test organisation (if it does not already exist):
+   1. Open the backoffice application
+   2. Create a new organisation with the name `e2e`. Do not forget to check "init with demo data"
+   3. Create a new admin user on this org with the email `admin@e2e.com`
+2. Start a local test environment:
+   1. start the Firebase emulator suite
+   2. start the backend normally
+   3. start the front-end application normally
```

Comment:
I first had to:
- go to packages/tests
- run `pnpm exec playwright install`
Now the tests are running but in error

I see a bunch of requests received by the backend 
<img width="987" alt="Capture d’écran 2024-09-05 à 21 47 22" src="https://github.com/user-attachments/assets/f155bd5a-ab58-40b6-aff2-a82c04d16312">

but then the tests times out 

```1) [chromium] › 001-user-signin-flow.test.ts:16:1 › Users can sign in with Google ────────────────

    Tearing down "firebase" exceeded the test timeout of 30000ms.

    Error: locator.click: Test ended.
    Call log:
      - waiting for getByLabel('Open menu for user jbe@zorg.com')


       at ../page-object-models/firebase-auth-emulator.ts:45

      43 |   async deleteUser(email: string) {
      44 |     await this.newPage(async (page) => {
    > 45 |       await page.getByLabel(`Open menu for user ${email}`).click();
         |                                                            ^
      46 |       await page.getByRole('menuitem', { name: 'Delete user' }).click();
      47 |       await page.getByRole('button', { name: 'Delete' }).click();
      48 |       await expect(

        at cb (/Users/pascal/Documents/marble/marble-frontend/packages/tests/page-object-models/firebase-auth-emulator.ts:45:60)
        at FirebaseAuthEmulatorPage.newPage (/Users/pascal/Documents/marble/marble-frontend/packages/tests/page-object-models/firebase-auth-emulator.ts:13:11)
        at FirebaseAuthEmulatorPage.deleteUser (/Users/pascal/Documents/marble/marble-frontend/packages/tests/page-object-models/firebase-auth-emulator.ts:44:5)
        at Object.firebase (/Users/pascal/Documents/marble/marble-frontend/packages/tests/fixtures/firebase.ts:49:7)
```

Reply:
Is there a particular project id that I need to run the firebase emulator with ?

Reply:
This is good, it answer one of my concerns: prevent `pnpm install` to be bloated by playwright "full" install. So doing a manual step feels like a good thing right now, I'll add it to the Readme.

Concerning the FB emulator, I use the same one we use for dev, but it shouldn be a problem (using `firebase emulators:start` from the backend working directory works on my side)

Are you up to date with origin ? Reading the error message, it tries to delete `jbe@zorg.com` but now, tests are relying on a separate user `admin@e2e.com` so it's weird.